util/changeref.py

```diff
@@ -20,7 +22,12 @@
 
 if new_ref_type not in ["ref", "any", "doc"]:
     print("Third arg not a valid ref type string.")
-    sys.exit()    
+    sys.exit()  
+
+redirect = "  %s: %s\n" %(old_path, new_path)
+
+with open('s3_website.yml', 'a') as file:
```

Comment:
This approach requires that the `s3_website` file never changes structure. What would happen if the redirects listing wasn't the last item in the file?

I *wonder* if we could split out the redirects to their own file. Or some other solution that minimizes this kind of risk.  (My first thought was to parse in the yaml, update it internally, and then write the whole file back out. That seems like overkill, maybe.)

Reply:
I have fixed this issue. Now the redirects line is searched for and a redirect is added below the redirects line when found. So, ``s3_website`` file can change structure without any issue.